app/models/agents/website_agent.rb

```diff
@@ -277,14 +279,24 @@ def extraction_type
       end).to_s
     end
 
-    def extract_each(doc, &block)
+    def use_namespaces?
+      if value = interpolated.key?('use_namespaces')
+        boolify(interpolated['use_namespaces'])
+      else
+        interpolated['extract'].none? { |name, extraction_details|
```

Comment:
The default behavior is to remove all namespaces, and the `use_namespaces` option is there to override it.